modules/swagger-codegen/src/main/resources/typescript-axios/api.mustache

```diff
@@ -28,22 +28,42 @@ export class {{classname}}Resource {
     public {{nickname}}({{#allParams}}{{^isQueryParam}}{{paramName}}{{^required}}?{{/required}}: Partial<{{{dataType}}}>, {{/isQueryParam}}{{/allParams}}{{#hasQueryParams}}query?: { {{#queryParams}}{{paramName}}{{^required}}?{{/required}}: {{dataType}}{{^-last}},{{/-last}} {{/queryParams}}}, {{/hasQueryParams}}axiosConfig?: AxiosRequestConfig): Promise<{{#returnType}}{{{returnType}}}{{/returnType}}{{^returnType}}{}{{/returnType}}> {
         const reqPath = '{{path}}'{{#pathParams}}
                     .replace('{' + '{{baseName}}}', String({{paramName}}{{#isDateTime}}.toISOString(){{/isDateTime}})){{/pathParams}};
-{{#hasFormParams}}
+    {{#hasFormParams}}
+        let hasFile = false;
+        {{#formParams}}
+            {{#isFile}}hasFile = true;
+            {{/isFile}}
```

Comment:
This code is a little funky, but comments in the PR I based these changes off of didn't offer alternative solutions: https://github.com/swagger-api/swagger-codegen/pull/4852/files#r104352794.